src/status_im2/contexts/emoji_picker/view.cljs

```diff
@@ -0,0 +1,200 @@
+(ns status-im2.contexts.emoji-picker.view
+  (:require [oops.core :as oops]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.blur :as blur]
+            [react-native.core :as rn]
+            [react-native.gesture :as gesture]
+            [react-native.platform :as platform]
+            [reagent.core :as reagent]
+            [status-im2.contexts.emoji-picker.constants :as constants]
+            [status-im2.contexts.emoji-picker.data :as emoji-picker.data]
+            [status-im2.contexts.emoji-picker.style :as style]
+            [status-im2.contexts.emoji-picker.utils :as emoji-picker.utils]
+            [utils.debounce :as debounce]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [utils.transforms :as transforms]))
+
+(defn- on-press-category
+  [id index active-category scroll-ref]
+  (reset! active-category id)
+  (some-> ^js @scroll-ref
+          (.scrollToIndex #js
+                           {:index    (emoji-picker.data/get-section-header-index-in-data index)
+                            :animated false})))
+
+(defn- handle-on-viewable-items-changed
+  [event active-category filtered-data]
+  (when-not @filtered-data
+    (let [viewable-item (-> (oops/oget event "viewableItems")
+                            transforms/js->clj
+                            first
+                            :item)
+          is-header?    (and (map? viewable-item) (:header? viewable-item))
+          section-key   (if is-header?
+                          (:id viewable-item)
+                          (emoji-picker.data/section-group (get-in (vec viewable-item) [0 :group])))]
+      (when (and (some? section-key) (not= @active-category section-key))
+        (reset! active-category section-key)))))
+
+(defn- get-item-layout
+  [_ index]
+  #js
+   {:length constants/get-item-height
+    :offset (* constants/get-item-height index)
+    :index  index})
+
+(defn- section-header
+  [{:keys [title]} _ _ {:keys [theme]}]
```

Comment:
is there any particular reason for the `_` `_` args here? is it not possible to just use one single map for the component? 🤔